frontend_lib/src/component/PopinFixed/PopinFixed.styl

```diff
@@ -284,6 +285,23 @@ responsiveColumnStyle()
     & span
       display none
   .wsContentGeneric
+    &__header:has(input)
+      display list-item
+      height fit-content
+      .dropdown
+        display inline-block
+        height 24px
+    .kanban__contentpage__header:has(input)
+        padding-bottom 60px
+    .logbook__contentpage__header:has(input)
+        padding-bottom 60px
```

Comment:
css rules specific to an app should be located in the app code, not in frontend_lib which must stay generic

Reply:
done